concepticondata/conceptlists/Desrochers-2010-330.tsv

```diff
@@ -245,11 +245,11 @@ Desrochers-2010-330-243	243	calar	put on	PUT ON	158	Verb	4.2843137254902	5	3
 Desrochers-2010-330-244	244	calcificar	calcify			Verb	3.08823529411765	10		
 Desrochers-2010-330-245	245	calentar	heat	HEAT UP	1341	Verb	6.41176470588235	8	31	6.74
 Desrochers-2010-330-246	246	cantar	sing	SING	1261	Verb	6.34313725490196	6	137	6.21
-Desrochers-2010-330-247	247	cascar	to crack			Verb	4.65686274509804	6	2	
+Desrochers-2010-330-247	247	cascar	to crack	SPLIT OPEN	3080	Verb	4.65686274509804	6	2	
 Desrochers-2010-330-248	248	catar	taste	TASTE (SOMETHING)	21	Verb	4.34313725490196	5	3	
 Desrochers-2010-330-249	249	cazar	hunting	HUNT	1435	Verb	5.30392156862745	5	43	5.2
 Desrochers-2010-330-250	250	cerrar	close	SHUT	1046	Verb	6.64705882352941	6	197	6.46
-Desrochers-2010-330-251	251	cobijar	shelter			Verb	3.2156862745098	7	6	
+Desrochers-2010-330-251	251	cobijar	shelter	HUT	1678	Verb	3.2156862745098	7	6	
```

Comment:
I'd unmap. The search for the Spanish gloss seems to refer to covering oneself with a blanket or similar.

Reply:
I agree.